BasicSample/Assets/Shared/Scripts/HideHandSpheresWhenNotActive.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR;
+
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    public class HideHandSpheresWhenNotActive : MonoBehaviour
```

Comment:
nit: could you add a comment here for why we need this script?
maybe something like "An XRPoseDriver will not update the visuals of their attached GameObjects when tracking is lost. This script enable or disables GameObjects according to the tracking state of their related XRNode."

Reply:
added

Reply:
Awesome! Could you add the 
`/// <summary>`
`/// </summary>`
tags on this comment too?

Reply:
Would this add into public documentation of any kind?

Reply:
usually the xml tags are for public documentation and from previous reviews with Oliver, he said we dont need them for sample code.

> > What do you mean by next to XRPoseDrivers? Do you have a way in mind?
> 
> I was thinking of a MonoBehaviour script which would be added to the same GameObject as the one with the TrackedPoseDriver
> 
> For a mockup, something that would look like this: ![image](https://user-images.githubusercontent.com/12822425/202782815-e87e3dad-f9a5-4a56-ad67-8577e5946f90.png)
> 
> Again, the current iteration is good to fix the problem in hand - I'm just thinking we can reuse this if we hit this problem in the future

This was my initial idea, but then since it is hand as you pointed out - i need to write 2 scripts (each for a hand) and felt what I have now with one script is better. But, totally agree for something in the future :)

Reply:
Got it - good to know!